kuma/wiki/templates/wiki/feed_docs_updated.html

```diff
@@ -0,0 +1,12 @@
+<p>
+  <a href="{{ obj.parent.get_absolute_url()|absolutify }}" title="{{ obj.parent.title }}">
+    {% trans locale=obj.parent.locale %}View {locale} parent{% endtrans %}
+  </a>
+  (<a href="{{ mod_url|absolutify }}">{% trans modified=obj.parent.modified %}last modified at {modified}{% endtrans %}</a>)
+</p>
+<p>
+  <a href="{{ add_utm(obj.get_edit_url()|absolutify, 'feed', medium='format') }}" title="{{ obj.title }}">
```

Comment:
`medium='format'` marks every pageview with the literal string `format`; we should change it to be either `rss` or `json`